src/feature/commands/index.ts

```diff
@@ -74,18 +74,16 @@ export async function checkForCommand() {
     core.getInput('repo-token', { required: true }),
   );
 
-  // Get all the members from the organization.
-  const allowedMembers = await client.rest.orgs.listMembers({
-    org: repo.owner,
-  });
-  if (allowedMembers.status !== 200) {
-    core.info('Failed to fetch the members from the organization');
-    return;
-  }
-  if (
-    !allowedMembers.data.find((member) => member.login === commentUser.login)
-  ) {
-    core.info('The comment author is not a organization member');
+  try {
+    const memberToken = core.getInput('member-token');
+    const memberClient = memberToken ? github.getOctokit(memberToken) : client;
+
+    await memberClient.rest.orgs.checkMembershipForUser({
```

Comment:
Does this throw an error if they're not a member, or do you need to actually check the response?